src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
 {
 }
 
+////////////////
+// User modes
+////////////////
+
+const QString & KviBasicIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'O': return __tr2qs("O: Local IRC operator (LOCOP)"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
+		case 'c': return __tr2qs("c: Recipient for cconn messages"); break;
+		case 'd': return __tr2qs("d: Recipient for server debug notices"); break;
+		case 'f': return __tr2qs("f: Recipient for full server notices"); break;
+		case 'i': return __tr2qs("i: Invisible"); break;
+		case 'k': return __tr2qs("k: Recipient for server KILL messages"); break;
+		case 'n': return __tr2qs("n: Recipient for nick changes"); break;
+		case 'o': return __tr2qs("o: IRC operator (OPER)"); break;
```

Comment:
Reason I prefer parenthesis is because it identifies it as a separate synonym whereas a hyphen denotes it as a type. For example, on Rizon we have SWHOIS strings that return "is an IRC Operator - Network Administrator" or "is an IRC Operator - Server Administrator". IMO a hyphen would identify a "type" of operator, whereas here we're just showing the user the short hand version that people generally know as well.